app/models/region.rb

```diff
@@ -53,6 +53,52 @@ def protected_areas_per_governance
     }
   end
 
+  def protected_areas_per_iucn_category
+    region_data = Hash.new { |hash, key| hash[key] = Hash.new }
+    processed_data = []
+    total_region_count = []
+
+    countries.each do |country|
+      country.protected_areas_per_iucn_category.each do |protected_area|
+        region_pa_category = region_data[protected_area["iucn_category_name"]]
+        region_pa_category["count"] ||= 0
+        region_pa_category["count"] += protected_area["count"].to_i
+        total_region_count << protected_area["count"].to_i
+        region_pa_category["percentage"] ||= 0
+        region_pa_category["percentage"] += protected_area["percentage"].to_f
```

Comment:
Seems like there would be an easier way to sum the counts rather than looping over everything and adding them up one by one. Perhaps inject? I'm not sure what ruby version we're on either, but 2.4 has this http://ruby-doc.org/core-2.4.0_rc1/Enumerable.html#method-i-sum

Reply:
Originally I tried to use `inject` in the first version of this method. But the code got really complex and had to be re-mapped again afterwards. So this was the rewrite of that. I will look into this suggestion @spencerldixon thanks!